etl/steps/data/garden/demography/2023-10-09/life_expectancy.meta.yml

```diff
@@ -0,0 +1,142 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  selected_entities:
+    with_americas: &sel_entities_americas
+      - Africa
+      - Americas
+      - Asia
+      - Europe
+      - Oceania
+      - World
+    standard: &sel_entities
+      - Africa
+      - North America
+      - Latin America and the Caribbean
+      - Asia
+      - Europe
+      - Oceania
+      - World
+  title_base: Period life expectancy
+  life_expectancy_0:
+    description_short: |-
+      The total number of years a newborn is expected to live.
+  common:
+    description_short: |-
+      <% if (age == 0) %>
+      {definitions.life_expectancy_0.description_short}
+      <% else %>
+      <% if (sex == 'female') %>
+      The total number of years a person is expected to live, among people who have already reached << age >> years of age.
+      <% elif (sex == 'male') %>
+      The total number of years a person is expected to live, among people who have already reached << age >> years of age.
```

Comment:
`among males`?

Reply:
I agree we could do this, but was unclear from discussions with Saloni, as it may sound strange. Will check with her again.

Reply:
Changing it, thanks!